src/UniqueMsg.chpl

```diff
@@ -164,6 +164,12 @@ module UniqueMsg
         return (permutation, segments);
       }
 
+      inline proc cleanup(str_names: [] string, st) throws {
+        forall name in str_names {
```

Comment:
I don't think the symbol table is parallel safe, so probably want this to be serial.

Reply:
@ronawho you are totally right. I have it written as a for, but did not save the file before pushing that is my bad. Should be updated now.